pages/DSNP/Announcements.md

```diff
@@ -42,6 +42,15 @@ For example, if a user creates a Reaction Announcement, they may remove that rea
 
 ## Non-Normative
 
+### Duplicate Announcements
+
+Due to the distributed nature of DSNP, duplicate Announcements are possible from time to time.
```

Comment:
Do we need to talk about how an end service can dedup these announcements or that's just not a protocol concern?